data/isdv4-10g.tablet

```diff
@@ -0,0 +1,15 @@
+# this is for the Wacom pen + touchscreen as found in the Fujitsu T935
```

Comment:
Have a look at e.g. commit ddc4e01ad61681764af06ef665f6c4dfaba7b3fe, it shows how the sysinfo is mentioned in the .tablet file so we have the link back to it when we need. Once that's in, the test will pass.

Reply:
I added the comment about the sysinfo file, committed and pushed it.  I hope this is what you meant me to do.  :-)  Thanks for all the help with this.
Rob